### DISCUSSION TITLE
Problem recorder after update

### DISCUSSION THREAD

**Discussion by cmsite** [home-assistant/frontend#7970] [Configuration] [answered]
Hi guys,

When I updated now it always gives these errors, how fix?

![image](https://user-images.githubusercontent.com/50043260/102069691-029c9f00-3df6-11eb-930d-7ec5587d78db.png)

![image](https://user-images.githubusercontent.com/50043260/102069772-1fd16d80-3df6-11eb-933e-2dbeb8de9474.png)

![image](https://user-images.githubusercontent.com/50043260/102070801-860ac000-3df7-11eb-8153-061cf0929251.png)

**Comment by spacegaier** [comment, marked as answer]
Did you check the logs? What do they say? Also: This does not seem to be a frontend issue.

**Reply by cmsite** [reply]
I understand that some configuration is missing:

https://user-images.githubusercontent.com/50043260/102070801-860ac000-3df7-11eb-8153-061cf0929251.png

**Reply by spacegaier** [reply]
So, it's purely a backed/core issue. Nothing the frontend team can do.

**Reply by cmsite** [reply]
how move?

**Reply by spacegaier** [reply]
You can't. Someone from the HA team needs to do that, although I am not sure if discussions can be moved at all (GitHub issues can).
